save-cosv/src/main/kotlin/com/saveourtool/save/cosv/repository/CosvRepository.kt

```diff
@@ -74,4 +75,14 @@ interface CosvRepository {
     fun deleteAll(
         identifier: String,
     ): Flux<LocalDateTime>
+
+    /**
+     * Get all COSV files (versions) with provided [identifier]
+     *
+     * @param identifier
+     * @return [Flux] with all COSV files versions
+     */
+    fun listVersions(
```

Comment:
suggestion to add also:
```kotlin
    /**
     * Downloads COSV from repository as [Flux] of [ByteBuffer] (stream)
     *
     * @param keyId
     * @return [Flux] of [ByteBuffer] with content of COSV
     */
    fun downloadAsStream(
        keyId: Long,
    ): Flux<ByteBuffer>
```

Reply:
implementation in `CosvRepositoryInStorage`
```kotlin
override fun downloadAsStream(keyId: Long): Flux<ByteBuffer> = cosvFileStorage.downloadByKeyId(keyId)
```
implementation in `CosvFileStorage`:
```kotlin
fun downloadByKeyId(keyId: Long): Flux<ByteBuffer> = blockingToMono { s3KeyManager.findKeyByEntityId(keyId) }
    .switchIfEmptyToNotFound {
        "Not found CosvFile by id $keyId"
    }
    .flatMapMany { download(it) }
```